CHANGELOG.adoc

```diff
@@ -14,6 +14,7 @@
 === Improvements
 
 - https://github.com/eclipse-sirius/sirius-components/issues/799[#799] [diagram] The buttons in the diagram's toolbar now have proper tooltips
+- https://github.com/eclipse-sirius/sirius-components/issues/773[#773] [compatibility] The synchronization policy of the node descriptions is now properly computed from the `AbstractNodeMapping`
```

Comment:
Do not forget changelog entries for the createView / deleteView support. We need to keep the changelog up to date to help consumers of Sirius Components and external contributors.

Reply:
Done